docs/docker/README.md

```diff
@@ -58,6 +58,35 @@ docker run -p 9000:9000 --name minio1 \
   minio/minio server /data
 ```
 
+### Run Minio Docker as non root user
+Release `RELEASE.2018-09-01T00-38-25Z` onwards Minio server runs as non-root within the container. However, this is applicable only if you're deploying new Minio instance (not upgrading from older releases). Deployments upgrading from older Minio versions and having data already stored on the backend, will continue to run as root. 
```

Comment:
Fix the release tag here... we can update this after the release.

Reply:
Update with `RELEASE.2018-10-06T00-15-16Z`